integration-cli/docker_cli_run_test.go

```diff
@@ -1349,6 +1349,8 @@ func TestRunAddingOptionalDevices(t *testing.T) {
 }
 
 func TestRunModeHostname(t *testing.T) {
```

Comment:
This is another case where we have a clearly cli-testing useful test bundled together with a host-dependent test.  Do you think there'd be value in splitting this one?